Makefile

```diff
@@ -170,3 +170,6 @@ graphql-docs:
 
 gqlgen-generate:
 	cd internal/graphql; go run -mod=mod github.com/99designs/gqlgen generate
+
+error:
+	go run github.com/layer5io/meshkit/cmd/errorutil -d . update
```

Comment:
@kumarabd I see that the PR is merged however we might want to exclude mesheryctl in here, if it is a separate component (as @lee indicated when he manually added some errors)?
There is a command line flag for this: `skip-dirs`.